### PR TITLE
Added files to create a service on the Raspberry Pi

### PR DESCRIPTION
**Pull request by sebastiengrd**
Useful to automatically run rocket code on startup. Needs to be updated in the future to restart if it crashes

### CODE REVIEW

**Review by ajayyy — APPROVED** [review]

File: control/service/rocket_startup.service [hunk 1]
```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=rocket_code_startup_script
+After=network.target
+
+[Service]
+ExecStart=sudo /home/pi/rocket-code-2020/sbgECom/bin/MainLoop &
```
Comment (jonapap) [inline control/service/rocket_startup.service:6]: Are you sure this is the right path? Shouldn't this be under some build folder?

**Review by ajayyy — CHANGES_REQUESTED** [review]
We should take the changes to the service files from the pi for this one too @sebastiengrd

**Review by ajayyy — APPROVED** [review]

File: CMakeLists.txt [hunk 2]
```diff
@@ -108,15 +108,14 @@ if(${CMAKE_SYSTEM_PROCESSOR} STREQUAL "arm")
 	target_link_directories(MainLoopLib PUBLIC "$ENV{LIBI2C_DIRECTORY}/lib")
 	
 else()
+	target_link_libraries(MainLoopLib systemd)
```
Comment (jonapap) [inline CMakeLists.txt:111]: Heads up that this is another dynamically linked library, so no support for cross-compile

**Review by jonapap — APPROVED** [review]

**Comment by ajayyy** [comment]
@jonapap good to merge this as well?

**Comment by jonapap** [comment]
> @jonapap good to merge this as well?

Yes, LGTM

**Pull request merged**